tools/mira3/mira.xml

```diff
@@ -21,32 +21,32 @@ mira
     ## Not easy in Galaxy to add sanger to --job, so use load_sequence_data(lsd) instead
     ## I expect hard trimmed FASTQ files with no NCBI traceinfo XML file
     -LR:lsd=1:mxti=0:ft=fastq -FN:fqi=${condSanger.filename}
-    ${condSanger.sanger_extra}
+    "${condSanger.sanger_extra}"
```

Comment:
Quotes will break the extra arbitrary values - they take the form of one or more new ``-arg=value`` settings.

Reply:
Yes, text parameters like ``--title "Some text here"`` work fine via ``--title "$TITLE"`` (although I think single quotes are preferred as in ``--title '$TITLE'`` which does not expand environment variables). The very fact they are quoted narrows the scope for security holes in the sanitisation.

Reply:
That is strange, but could happen if MIRA v3 does its own command line parsing rather than using a standard library?